gallery/src/main.scss

```diff
@@ -61,3 +61,12 @@
 
   position: relative;
 }
+
+.gallery-example-image {
+  height: auto;
```

Comment:
Necessary?

Reply:
> Necessary?

Not really, just habit: https://github.com/sonatype/sonatype-react-shared-components/pull/28/commits/8a9c70da52cbaa5a410297b32f848f1439cdcf02